cmd/cleanup-controller/handlers/cleanup/handlers.go

```diff
@@ -170,9 +170,10 @@ func (h *handlers) executePolicy(ctx context.Context, logger logr.Logger, policy
 					if err := h.client.DeleteResource(ctx, resource.GetAPIVersion(), resource.GetKind(), namespace, name, false); err != nil {
 						debug.Error(err, "failed to delete resource")
 						errs = append(errs, err)
+						h.createEvent(policy, resource, corev1.EventTypeWarning, err)
 					} else {
 						debug.Info("deleted")
-						h.createEvent(policy, resource)
+						h.createEvent(policy, resource, corev1.EventTypeNormal, nil)
```

Comment:
Can't we just use `err` to determine event type ?

Reply:
Good idea, will update in short.